### PR TITLE
Support for batch exists calls

### PR DESCRIPTION
**Pull request by yaronel**
Similar to `batch-get` only returns booleans rather than the actual records.
Also added missing `batch-get` function to the mock.

### CODE REVIEW

File: src/aerospike_clj/client.clj [hunk 1]
```diff
@@ -291,6 +298,27 @@
...
+  "Check for each key in a batch if it exists. `indices` is a collection of maps
+  of the form `{:index \"key-name\" :set \"set-name\"}`. The result is a
+  vector of booleans in the same order as indices."
+  ([db indices]
+   (exists-batch db indices {}))
+  ([db indices conf]
+   (let [client (get-client db)
+         op-future (d/deferred)
+         start-time (System/nanoTime)
+         indices (utils/v->array Key (mapv #(create-key (:index %) (:dbns db) (:set %)) indices))]
```
Comment (barkanido) [inline src/aerospike_clj/client.clj]: might be worthwhile to bind `(:dbns db)` in the `let` vector

File: src/aerospike_clj/client.clj [hunk 2]
```diff
@@ -291,6 +298,27 @@
...
+     (.exists ^AerospikeClient client
+              ^EventLoop (.next ^NioEventLoops (:el db))
+              (reify-exists-array-listener op-future)
+              ^BatchPolicy (:policy conf)
+              ^"[Lcom.aerospike.client.Key;" indices)
+     (let [d (d/chain' op-future
+                       #(vec %)
```
Comment (barkanido) [inline src/aerospike_clj/client.clj]: isn't `vec` enough?

File: src/aerospike_clj/client.clj [hunk 3]
```diff
@@ -291,6 +298,27 @@
...
+                       (:transcoder conf identity))]
+       (register-events d db "read-batch" nil start-time)))))
```
Comment (barkanido) [inline src/aerospike_clj/client.clj:321]: do we want a "exists-batch" metric?

Comment (yaronel) [reply src/aerospike_clj/client.clj:321]: I personally don't think we need this granularity, but your call.

**Review by barkanido — APPROVED** [review]

**Pull request merged**